tests/Unit/Protocol/ProtocolTestCase.php

```diff
@@ -90,11 +90,11 @@ public function testAckVersionTwo()
         $instance = $this->getProtocol(Version::VERSION_1_2);
 
         $actual = $instance->getAckFrame(
-            new Frame(null, ['message-id' => 'id-value', 'ack' => 'ack-id-value']),
```

Comment:
are these test-changes related to the type changes, or could be separated because unrelated?